Backend/controllers/seller.controller.js

```diff
@@ -351,6 +353,60 @@ const getAllDeals = asyncHandler(async (req, res) => {
         )
 })
 
+const updateOrCreateSellerInformation = asyncHandler(async (req, res) => {
+    const { email, phoneNumber, address, tags } = req.body;
+
+    // Validate input
+    if (!email || !phoneNumber) {
+        throw new ApiError(400, "Email and Phone Number are required.");
+    }
+
+    // Check if the seller exists based on email
+    let seller = await SellerDetails.findOne({ email });
+
+    if (seller) {
```

Comment:
If seller is not found, just throw an error

Reply:
required modifications is done.